src/lotus_json/mod.rs

```diff
@@ -330,7 +330,7 @@ pub mod hexify {
         T: Num + std::fmt::LowerHex,
         S: Serializer,
     {
-        serializer.serialize_str(format!("0x{value:x}").as_str())
+        serializer.serialize_str(format!("{value:#x}").as_str())
```

Comment:
https://doc.rust-lang.org/stable/std/fmt/trait.LowerHex.html

Reply:
We could do same in `eth_block_number`.

Reply:
Fixed.